### ISSUE TITLE
multilanguage support

### ISSUE THREAD

**Issue by Kikobeats** [balderdashy/waterline#388]
Hello!

Love waterline and validations in models. However I would love it even more if errors validations messages could be multi-language.

Using config/locales files, do you think that it can be possible?

**Comment by mikermcneil** [comment]
Thanks for posting, @Kikobeats.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/waterline/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
